uds/database/data_record/text_data_record.py

```diff
@@ -0,0 +1,87 @@
+"""Definition of TextDataRecord which is class for encode and decode values of data records."""
+
+__all__ = ["TextDataRecord"]
+
+from typing import Dict, Optional
+
+from .abstract_data_record import AbstractDataRecord, DataRecordPhysicalValueAlias, DecodedDataRecord
+from .raw_data_record import RawDataRecord
+
+
+class TextDataRecord(RawDataRecord, AbstractDataRecord):
```

Comment:
please, call it `TextTableDataRecord` (cause we are going to have one more text data record)